docs/src/content/docs/commands/GETEX.md

```diff
@@ -45,50 +51,57 @@ The `GETEX` command can raise errors in the following scenarios:
 3. `Invalid expiration time`: If the expiration time is not a valid integer or timestamp, it will return an error.
    - Error message: `(error) ERR value is not an integer or out of range`
 
-## Example Usage
+## Examples
```

Comment:
Let's keep it consistent with what we have in other docs. In future, we can rename all of them in one PR to make them consistent. 
```suggestion
## Example Usage
```